src/overlays/ovl_i2/fox_me.c

```diff
@@ -1,97 +1,2527 @@
+    }
+
+    if (arg1 < 4) {
+        Matrix_RotateY(gCalcMatrix, boss->obj.rot.y * M_DTOR, 0);
+        Matrix_RotateX(gCalcMatrix, boss->obj.rot.x * M_DTOR, 1);
+        Matrix_RotateZ(gCalcMatrix, boss->obj.rot.z * M_DTOR, 1);
+        Matrix_RotateZ(gCalcMatrix, arg1 * (M_PI / 2), 1);
+
+        src.x = 0.0f;
+        src.y = 200.0f;
+        src.z = 500.0f;
+
+        Matrix_MultVec3fNoTranslate(gCalcMatrix, &src, &dest);
+
+        func_8007D2C8(boss->obj.pos.x + dest.x, boss->obj.pos.y + dest.y, boss->obj.pos.z + dest.z, 10.0f);
+        func_8007BFFC(boss->obj.pos.x + dest.x, boss->obj.pos.y + dest.y, boss->obj.pos.z + dest.z, 0.0f, 0.0f, 0.0f,
+                      5.0f, 15);
+    }
+}
+
+void func_i2_80189624(void) {
+    if (gGameFrameCount & 1) {
+        D_801784B8 = D_801784C4 = 0.0f;
+        D_801784BC = D_801784C8 = 5.0f;
+        D_801784C0 = D_801784CC = 0.0f;
+        D_801784EC = D_801784F8 = 0.0f;
+        D_801784F0 = D_801784FC = 100.0f;
+        D_801784F4 = D_80178500 = 0.0f;
+    } else {
+        D_801784B8 = D_801784C4 = 0.0f;
+        D_801784BC = D_801784C8 = -5.0f;
+        D_801784C0 = D_801784CC = 0.0f;
+        D_801784EC = D_801784F8 = 0.0f;
+        D_801784F0 = D_801784FC = 110.0f;
+        D_801784F4 = D_80178500 = 0.0f;
+    }
+}
+
+void func_i2_8018978C(Boss* boss) {
+    f32 sp7C;
+    f32 rand;
+    s32 i;
+    s32 var_v0;
+
+    Vec3f src;
+    Vec3f dest;
+
+    gBossFrameCount++;
+
+    Matrix_RotateY(gCalcMatrix, boss->obj.rot.y * M_DTOR, 0);
+    Matrix_RotateX(gCalcMatrix, boss->obj.rot.x * M_DTOR, 1);
+    Matrix_RotateZ(gCalcMatrix, boss->obj.rot.z * M_DTOR, 1);
+
+    if (gGameFrameCount & 16) {
+        boss->swork[19] += 32;
+        if (boss->swork[19] > 128) {
+            boss->swork[19] = 128;
+        }
+    } else {
+        boss->swork[19] -= 32;
+        if (boss->swork[19] < 0) {
+            boss->swork[19] = 0;
+        }
+    }
+
+    if (!(gGameFrameCount & 56)) {
+        boss->fwork[3] = D_i2_80195534[gGameFrameCount & 7];
+        boss->fwork[4] = D_i2_80195534[gGameFrameCount & 7];
+        boss->fwork[5] = D_i2_80195534[gGameFrameCount & 7];
+        boss->fwork[6] = D_i2_80195534[gGameFrameCount & 7];
+    }
+
+    if (!((gGameFrameCount + 20) & 56)) {
+        boss->fwork[7] = D_i2_80195534[(gGameFrameCount + 20) & 7];
+        boss->fwork[8] = D_i2_80195534[(gGameFrameCount + 20) & 7];
+    }
+
+    if (boss->swork[9] != 0) {
+        boss->swork[9]--;
+    }
+
+    if (boss->swork[10] != 0) {
+        boss->swork[10]--;
+    }
+
+    if (boss->swork[11] != 0) {
+        boss->swork[11]--;
+    }
+
+    if (boss->swork[12] != 0) {
+        boss->swork[12]--;
+    }
+
+    if (boss->swork[13] != 0) {
+        boss->swork[13]--;
+    }
+
+    if (boss->swork[14] != 0) {
+        boss->swork[14]--;
+    }
+
+    if (boss->swork[15] != 0) {
+        boss->swork[15]--;
+    }
+
+    if (boss->swork[0] >= 6) {
```

Comment:
```suggestion
    if (boss->swork[0] > 5) {
```
Since 5 is checked against below, seems to me this should match it.